docs/rst/developer_manual/installation/sources/windows.rst

```diff
@@ -0,0 +1,359 @@
+.. include:: ../../../exports/alias.include
+.. include:: ../../../exports/roles.include
+
+.. _developer_manual_installation_sources_windows:
+
+#################################
+Windows installation from sources
+#################################
+
+The instructions for installing the |ddsrouter| application from sources and its required
+dependencies are provided in this page.
+It is organized as follows:
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 2
+
+
+Dependencies installation
+=========================
+
+|ddsrouter| depends on *eProsima Fast DDS* library and certain Debian packages.
```

Comment:
In general:
I would suggest to create a shortcut for FastDDS as it is for `|ddsrouter|`, so it always keeps the same format

Reply:
I agree but it won't go in this PR